src/main/java/com/hillel/items_exchange/service/UserService.java

```diff
@@ -26,18 +29,26 @@
 import static com.hillel.items_exchange.mapper.UserMapper.convertDto;
 import static com.hillel.items_exchange.mapper.UtilMapper.convertAllTo;
 import static com.hillel.items_exchange.mapper.UtilMapper.convertToDto;
+import static com.hillel.items_exchange.model.enums.Status.ACTIVE;
+import static com.hillel.items_exchange.model.enums.Status.DELETED;
 import static com.hillel.items_exchange.util.Collections.extractAll;
 import static com.hillel.items_exchange.util.MessageSourceUtil.getMessageSource;
+import static com.hillel.items_exchange.util.MessageSourceUtil.getParametrizedMessageSource;
+import static java.time.temporal.ChronoUnit.DAYS;
 
 @Service
 @RequiredArgsConstructor
 public class UserService {
 
+    public static final String ONE_TIME_PER_DAY_AT_3_AM = "0 0 3 * * * ";
```

Comment:
Please put it in the property file and use this field with @Value.

Reply:
@Scheduled(cron = ONE_TIME_PER_DAY_AT_3_AM)
![image](https://user-images.githubusercontent.com/49348448/109512743-15497a80-7aad-11eb-8eb6-36978dc882f3.png)